package.json

```diff
@@ -5,8 +5,8 @@
   "main": "src/server.js",
   "scripts": {
     "postinstall": "concurrently \"cd client && npm install\" \"cd api && npm install\"",
-    "start": "concurrently \"cd api && node src/server.js\" \"cd client && npm start\"",
-    "dev": "concurrently \"cd api && node src/server.js\" \"cd client && npm start\""
+    "start": "concurrently \"cd api && node src/index.js\" \"cd client && npm start\"",
+    "dev": "concurrently \"cd api && node src/index.js\" \"cd client && npm start\""
```

Comment:
Is there any reason we are changing it?

Reply:
It's the change from [PR#233](https://github.com/Researchify/Researchify/pull/233) to fix the `npm run start` in researchify root folder. I believe changes in this file will not be shown after that PR is merged.

Reply:
Yeah I just changed it so I could run the application using the single command but PR #233 has now been merged so this change here isn't regarding anything from this PR.